Add a comment regarding read/write control via HTTP verbs #863 (#7909)
opensearch-project/documentation-website: _security/access-control/rest-layer-authz.md

The benefits of using the REST layer for authorization include the ability to authorize requests at the REST layer and filter out unauthorized requests. As a result, this decreases the processing burden on the transport layer while allowing granular control over access to APIs.

> Some read operations manage state (for example [scroll]({{site.url}}{{site.baseurl}}/api-reference/scroll/). It is, therefore, recommended to control read and write access using Security plugin [permissions]({{site.url}}{{site.baseurl}}/security/access-control/permissions/), instead of allowing/blocking HTTP request verbs.

Reviewer: ```suggestion
Some read operations, such as  [scroll]({{site.url}}{{site.baseurl}}/api-reference/scroll/), manage state. Therefore, it is recommended to control read and write access using the Security plugin [permissions]({{site.url}}{{site.baseurl}}/security/access-control/permissions/), instead of allowing/blocking HTTP request verbs.
```
